.github/workflows/release.yaml

```
@@ -0,0 +1,35 @@
+---
+name: Release
+on:
+  push:
+    branches:
+      - main
+  workflow_dispatch:
+
+jobs:
+  release-please:
+    uses: ./.github/workflows/release-please.yaml
+    secrets: inherit
+    permissions:
+      contents: write
+      pull-requests: write
+
+  tag-major-minor:
+    runs-on: ubuntu-latest
+    needs: release-please
+    permissions:
+      contents: write
+    if: ${{ needs.release-please.outputs.releases_created }}
+    steps:
+      - uses: actions/checkout@8ade135a41bc03ea155e62e844d188df1ea18608 # v4.1.0
+      - run: |
+          git config user.name github-actions[bot]
+          git config user.email 41898282+github-actions[bot]@users.noreply.github.com
+          git tag -d v${{ steps.release.outputs.major }} || true
+          git tag -d v${{ steps.release.outputs.major }}.${{ steps.release.outputs.minor }} || true
+          git push origin :v${{ steps.release.outputs.major }} || true
+          git push origin :v${{ steps.release.outputs.major }}.${{ steps.release.outputs.minor }} || true
```

```suggestion
          git tag --delete v${{ steps.release.outputs.major }} || true
          git tag --delete v${{ steps.release.outputs.major }}.${{ steps.release.outputs.minor }} || true
          git push --delete origin v${{ steps.release.outputs.major }} || true
          git push --delete origin v${{ steps.release.outputs.major }}.${{ steps.release.outputs.minor }} || true
```

To make it a little bit more readable